yarn-project/txe/src/txe_service/txe_service.ts

```diff
@@ -622,10 +622,15 @@ export class TXEService {
     return toForeignCallResult(arrayToBoundedVec(bufferToU8Array(plaintextBuffer), ciphertextBVecStorage.length));
   }
 
-  async getSharedSecret(address: ForeignCallSingle, ephPk: ForeignCallArray) {
+  async getSharedSecret(
+    address: ForeignCallSingle,
+    ephPKField0: ForeignCallSingle,
+    ephPKField1: ForeignCallSingle,
+    ephPKField2: ForeignCallSingle,
```

Comment:
This was a bug (the oracle didn't get used before this PR). Opening a new PR for such a small change felt excessive.